visualc/VS2010/mbedTLS.vcxproj

```diff
@@ -224,6 +224,11 @@
     <ClInclude Include="..\..\include\mbedtls\x509_crt.h" />
     <ClInclude Include="..\..\include\mbedtls\x509_csr.h" />
     <ClInclude Include="..\..\include\mbedtls\xtea.h" />
+    <ClInclude Include="..\..\crypto\include\psa\crypto.h" />

```

Comment:
TODO: Don't modify the vcxproj files like this. Instead, add a new vcxproj file with crypto in it, modifying the vcxproj generating scripts to make it.

Reply:
Our tests want to run generate and see no changes, but to do that requires they know whether the submodule should be used or not. This will no longer be an issue when the submodule is always used, so I'd like to suggest we defer providing a crypto submodule using vcxproj until then.

Note this isn't just for PSA either, as when one uses the crypto submodule, they'd like to have the header files from the crypto submodule, not from mbedtls, whether or not they want PSA enabled.